gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Mon Aug 22 17:36:22 EDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
```

Comment:
In the interest of using purely Java 8, you should probably keep to Gradle 6.9.x.

Reply:
Done in 05982a1accc1c5cf88850673cb6244e36d53789f